lib/kvs/kvs_rparams.c

```diff
@@ -269,7 +270,7 @@ static const struct param_spec pspecs[] = {
         },
         .ps_bounds = {
             .as_uscalar = {
-                .ps_min = 0,
+                .ps_min = MPOOL_MBLOCK_SIZE_DEFAULT,
```

Comment:
.ps_min = 32\*1024

Reply:
What happens at runtime if this param is set to an enormous value?

Reply:
Actually, this should be:
.ps_min = 1024 * 32;
.ps_max = 1024 * 2048;